spec/PeriodicBackgroundSync-index.bs

```diff
@@ -70,28 +72,47 @@ The <code>periodicsync</code> event is considered to run <dfn>in the background<
 
 The user agent is considered to be [=online=] if the user agent has established a network connection. A user agent MAY use a stricter definition of being [=online=]. Such a stricter definition MAY take into account the particular [=service worker=] or origin a [=periodicsync registration=] is associated with.
 
-#TODO: link to PeriodicSyncManager.register().
-A <dfn>periodic Background Sync opportunity</dfn> allows periodic synchronization between the server and the web app, the exact interval of which is decided by the user agent.
+A <dfn>periodic Background Sync opportunity</dfn> allows periodic synchronization between the server and the web app, the exact interval of which is decided by the user agent. This can be requested through {{PeriodicSyncManager/register()}}.
 
 Constructs {#constructs}
 =========================
 
-A [=service worker registration=] has an associated <dfn>list of periodicsync registrations</dfn> whose element type is a [=periodicsync registration=].
+A [=/service worker registration=] has an associated <dfn>list of periodicsync registrations</dfn> whose element type is a [=periodicsync registration=].
 
-A <dfn>periodicsync registration</dfn> is a tuple consisting of a [=tag=], a dictionary of options, and a [=registration state=].
+A <dfn>periodicsync registration</dfn> is a tuple consisting of:
+* a [=periodicsync registration/service worker registration=] (a [=/service worker registration=]),
```

Comment:
These members should be `dfn`s, not links

Reply:
I have combined the two lists.